test/commons.ml

```diff
@@ -16,37 +17,65 @@ module Commons = struct
     | Error   of string
   [@@deriving bin_io, show { with_path = false }]
 
+  type message =
```

Comment:
Can't we reuse `message.ml` 🤔 ?

Reply:
Consumer should have the freedom to define their own type for messages sent back and forth between nodes, this is an example of that. Message.t has peer information that the consumer shouldn't necessarily have to include in their messages.

Reply:
Yeah, sorry, I was mixed `payload` and `label`. My bad.